### PR TITLE
upd(element-desktop-deb): `1.10.4` -> `1.10.7`

### PR DESCRIPTION
**Pull request by cat-master21**
Notice that I needed to change the gives because of:

>pumpkinking@pumpkinking-desktop:~$ pacstall -Up
>
>[sudo] password for pumpkinking: 
>[+] INFO: Checking for updates
>[+] INFO: Packages can be upgraded
>Upgradable: 2
>	element-desktop-deb @ pacstall/pacstall-programs 
>	github-desktop @ pacstall/pacstall-programs 
>
>Do you want to upgrade element-desktop-deb? [Y/n] y
>Do you want to view/edit the pacscript [y/N] n
>[+] INFO: Sourcing pacscript
>[+] INFO: URL exists
>[!] ERROR: element-desktop-deb breaks element-desktop, which is currently installed by apt
>[+] INFO: Cleaning up
>Do you want to upgrade github-desktop? [Y/n] y

### CODE REVIEW

File: packages/element-desktop-deb/element-desktop-deb.pacscript [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name="element-desktop-deb"
-gives="element-desktop"
-version="1.10.4"
-breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
+gives="element-desktop-deb"
+version="1.10.6"
+breaks="$element-desktop-deb ${gives}-git ${gives}-bin ${gives}-app"
```
Comment (Henryws) [inline packages/element-desktop-deb/element-desktop-deb.pacscript:4]: `$element-desktop-deb` is not a variable

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/element-desktop-deb/element-desktop-deb.pacscript [hunk 2]
```diff
@@ -1,8 +1,8 @@
 name="element-desktop-deb"
-gives="element-desktop"
-version="1.10.4"
-breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
+gives="element-desktop-deb"
```
Comment (Henryws) [inline packages/element-desktop-deb/element-desktop-deb.pacscript:2]: gives should be `element-desktop`

**Review by Henryws — CHANGES_REQUESTED** [review]

**Review by wizard-28 — DISMISSED** [review]

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/element-desktop-deb/element-desktop-deb.pacscript [hunk 3]
```diff
@@ -1,8 +1,8 @@
 name="element-desktop-deb"
 gives="element-desktop"
-version="1.10.4"
-breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
+version="1.10.6"
+breaks="element-desktop ${gives}-git ${gives}-bin ${gives}-app"
```
[the remark this answers is not in the document]
Comment (cat-master21) [reply packages/element-desktop-deb/element-desktop-deb.pacscript:4]: If I do that updating will fail!

[the remark this answers is not in the document]
Comment (Henryws) [reply packages/element-desktop-deb/element-desktop-deb.pacscript:4]: Ok, then add it to `replace`

**Comment by cat-master21** [comment]
Ready for review @Henryws

File: packages/element-desktop-deb/element-desktop-deb.pacscript [hunk 4]
```diff
@@ -1,8 +1,10 @@
 name="element-desktop-deb"
 gives="element-desktop"
-version="1.10.4"
-breaks="${gives} ${gives}-git ${gives}-bin ${gives}-app"
+version="1.10.7"
+replaces="element-desktop"
+repology="element-desktop"
```
Comment (wizard-28) [inline packages/element-desktop-deb/element-desktop-deb.pacscript:5]:
```suggestion
repology=("project: element-desktop")
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**